.gitignore

```diff
@@ -1 +1,3 @@
-*build
\ No newline at end of file
+*build
+*out
+CMakeUserPresets.json
```

Comment:
Would this entry better reside in a global .gitignore file (like where `.*.swp` would go)?

Reply:
We could, but then you rely on the developer to have that set up properly.